### PR TITLE
Debian package support

### PR DESCRIPTION
**Pull request by Arbuzov**
Add travis script to prepare debian package and deploy them to packageio

### CODE REVIEW

File: tests/dbTestCase.php [hunk 1]
```diff
@@ -16,8 +16,8 @@ protected function setUp()
     $conf = parse_ini_file(__DIR__.'/config.ini');
 
     // Make sure we have some clean, temporary output dir
-    exec( "rm -rf " . escapeshellarg($conf['savedir']) );
-    @mkdir( $conf['savedir'], 0777, TRUE );
+    @rmdir($conf['savedir']);
```
Comment (byo) [inline tests/dbTestCase.php]: rmdir will fail to remove the directory if it's not empty, that's why it was rm -rf before

File: deploy/DEBIAN/changelog [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (byo) [inline deploy/DEBIAN/changelog]: Shouldn't we have some first version info here ?

File: tests/dbTestCase.php (hunk 1 above)
Comment (Arbuzov) [inline tests/dbTestCase.php]: let me think... 
I can`t proove but something wrong to use OS specific command

**Comment by idler** [comment]
Awesome!
14 июля 2015 г. 20:01 пользователь "Arbuzov Sergey" <
notifications@github.com> написал:

**Pull request merged**